### PR TITLE
Added support for WINAPI_PARTITION_TV_TITLE to support XboxOne applications

### PR DESCRIPTION
**Pull request by gaspardpetit**
It was easiest to reuse the GTEST_OS_WINDOWS_PHONE macro since XboxOne titles have very similar constraints.

### CODE REVIEW

**Comment by gennadiycivil** [comment]
Thank you for your contribution. Please provide test cases / testing evidence showing the positive run

**Comment by gaspardpetit** [comment]
Do you have a suggestion on how to do this? Keep in mind that I cannot bundle the Microsoft XDK in my tests.

File: googletest/include/gtest/internal/gtest-port-arch.h [hunk 1]
```diff
@@ -54,6 +54,9 @@
 #   define GTEST_OS_WINDOWS_PHONE 1
 #  elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP)
 #   define GTEST_OS_WINDOWS_RT 1
+#  elif WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_TV_TITLE)
+#   define GTEST_OS_WINDOWS_PHONE 1
```
Comment (gennadiycivil) [inline googletest/include/gtest/internal/gtest-port-arch.h]: Why are we defining GTEST_OS_WINDOWS_PHONE 1 ? I see that the other definition, GTEST_OS_WINDOWS_TV_TITLE is used in line 1664, gtest.cc but I dont see GTEST_OS_WINDOWS_PHONE used anywhere... could you please explain

**Comment by gennadiycivil** [comment]
As far as the test run, just provide your test output to show how this works.

**Comment by gennadiycivil** [comment]
Ping

**Comment by gaspardpetit** [comment]
I will get back to you with test outputs shortly - I just haven't had a free minute to do it in the last days

**Comment by gennadiycivil** [comment]
Ping

**Comment by gaspardpetit** [comment]
Hello, I would have some time to submit this again with results, but to do so, I need https://github.com/google/googletest/pull/1201 to be integrated - `printf` goes nowhere on the xboxone, with the submited pull request I can route the `printf`s to `OutputDebugString` easily. Also, could you tell me which unit tests you expect to see passing? Many don't compile because of file system access and environment variables, but I assume this is also the case for Windows Phone

**Comment by gaspardpetit** [comment]
I was able to run the following tests:

		//#include "test/gtest-filepath_test.cc"
		#include "test/gtest-linked_ptr_test.cc"
		#include "test/gtest-message_test.cc"
		//#include "test/gtest-options_test.cc"
		//#include "test/gtest-port_test.cc"
		#include "test/gtest_pred_impl_unittest.cc"
		#include "test/gtest_prod_test.cc"
		#include "test/gtest-test-part_test.cc"
		#include "test/gtest-typed-test_test.cc"
		#include "test/gtest-typed-test2_test.cc"
		//#include "test/gtest_unittest.cc"
		#include "test/production.cc"

The others required `getenv`, `ChDir` or `CaptureStdout`

Would this output be enough to get approval for the pull request?

		Connect to pipe \SEVPipe\9021f601f657ec5f
		[==========] 
		Running 239 tests from 46 test cases.
		[----------] 
		Global test environment set-up.
		[----------] 
		1 test from TestPartResultArrayDeathTest

		[ RUN      ] 
		TestPartResultArrayDeathTest.DiesWhenIndexIsOutOfBound

		[       OK ] 
		TestPartResultArrayDeathTest.DiesWhenIndexIsOutOfBound
		 (0 ms)
		[----------] 
		1 test from TestPartResultArrayDeathTest (0 ms total)
		[----------] 
		4 tests from TypedTestCasePStateDeathTest

		 (1 ms)
		[----------] 
		MessageTest.ConstructsFromCString
		[       OK ] 
		MessageTest.StreamsNullCString
		MessageTest.StreamsNULChar
		MessageTest.StreamsBasicIoManip
		MessageTest.GetString
		 (0 ms)
		EXPECT_PRED1Test.FunctionOnBuiltInTypeSuccess
		[ RUN      ] 
		[       OK ] 

		[ RUN      ] 

		[       OK ] 
		[       OK ] 
		8 tests from EXPECT_PRED_FORMAT1Test
		[ RUN      ] 

		[ RUN      ] 
		[----------] 
		ASSERT_PRED_FORMAT1Test.FunctionOnUserTypeSuccess
		[       OK ] 
		[ RUN      ] 
		[----------] 
		[       OK ] 
		[       OK ] 

		[       OK ] 

		[       OK ] 
		ASSERT_PRED2Test.FunctionOnUserTypeSuccess
		 (1 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctorOnBuiltInTypeSuccess

		[       OK ] 
		ASSERT_PRED2Test.FunctorOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctorOnUserTypeSuccess

		[       OK ] 
		ASSERT_PRED2Test.FunctorOnUserTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctionOnBuiltInTypeFailure

		[       OK ] 
		ASSERT_PRED2Test.FunctionOnBuiltInTypeFailure
		 (1 ms)
		[ RUN      ] 
		 (0 ms)
		[ RUN      ] 
		[       OK ] 
		EXPECT_PRED3Test.FunctionOnUserTypeFailure

		8 tests from ASSERT_PRED3Test (7 ms total)
		EXPECT_PRED_FORMAT3Test.FunctorOnUserTypeFailure
		ASSERT_PRED_FORMAT3Test.FunctorOnBuiltInTypeFailure

		ASSERT_PRED_FORMAT3Test.FunctorOnUserTypeFailure

		[       OK ] 
		ASSERT_PRED_FORMAT3Test.FunctorOnUserTypeFailure

		[       OK ] 
		EXPECT_PRED4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		EXPECT_PRED4Test.FunctionOnUserTypeSuccess

		[       OK ] 
		[ RUN      ] 
		EXPECT_PRED4Test.FunctorOnBuiltInTypeSuccess
		EXPECT_PRED4Test.FunctionOnBuiltInTypeFailure
		EXPECT_PRED4Test.FunctorOnBuiltInTypeFailure
		 (2 ms)
		8 tests from EXPECT_PRED4Test (9 ms total)
		[----------] 
		8 tests from ASSERT_PRED4Test

		[ RUN      ] 

		[       OK ] 
		ASSERT_PRED4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED4Test.FunctionOnUserTypeSuccess

		[       OK ] 
		[ RUN      ] 
		ASSERT_PRED4Test.FunctorOnUserTypeSuccess
		[       OK ] 
		ASSERT_PRED4Test.FunctionOnUserTypeFailure
		[ RUN      ] 
		8 tests from ASSERT_PRED4Test (8 ms total)
		8 tests from EXPECT_PRED_FORMAT4Test

		[       OK ] 
		 (0 ms)
		[ RUN      ] 
		EXPECT_PRED_FORMAT4Test.FunctionOnBuiltInTypeFailure
		[ RUN      ] 

		8 tests from EXPECT_PRED_FORMAT4Test (5 ms total)

		[       OK ] 
		ASSERT_PRED_FORMAT4Test.FunctionOnBuiltInTypeSuccess
		ASSERT_PRED_FORMAT4Test.FunctionOnUserTypeSuccess
		ASSERT_PRED_FORMAT4Test.FunctorOnUserTypeSuccess

		 (1 ms)
		ASSERT_PRED_FORMAT4Test.FunctorOnUserTypeFailure



		[       OK ] 
		 (2 ms)

		EXPECT_PRED5Test.FunctorOnUserTypeFailure

		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED5Test.FunctionOnUserTypeFailure

		[       OK ] 
		ASSERT_PRED5Test.FunctionOnUserTypeFailure
		8 tests from ASSERT_PRED5Test (10 ms total)
		EXPECT_PRED_FORMAT5Test.FunctionOnBuiltInTypeSuccess
		EXPECT_PRED_FORMAT5Test.FunctorOnBuiltInTypeSuccess
		EXPECT_PRED_FORMAT5Test.FunctorOnUserTypeSuccess
		[ RUN      ] 
		 (1 ms)
		8 tests from EXPECT_PRED_FORMAT5Test (6 ms total)
		ASSERT_PRED_FORMAT5Test.FunctionOnBuiltInTypeSuccess
		ASSERT_PRED_FORMAT5Test.FunctionOnUserTypeSuccess
		[ RUN      ] 
		ASSERT_PRED_FORMAT5Test.FunctionOnBuiltInTypeFailure
		 (1 ms)


		[ RUN      ] 

		[       OK ] 
		[       OK ] 
		TestPartResultTest.Passed
		TestPartResultTest.Failed
		 (0 ms)
		3 tests from TestPartResultArrayTest
		TestPartResultArrayTest.ContainsGivenResultAfterAppend
		[       OK ] 

		 (0 ms)
		 (0 ms)
		CommonTest/1.ValuesAreCorrect
		[       OK ] 
		, where TypeParam = int
		1 test from TypedTest2/0
		1 test from TypedTest2/0 (0 ms total)
		 (0 ms)
		NumericTest/1.DefaultIsZero
		[ RUN      ] 
		TypedTestCasePStateTest.IgnoresOrderAndSpaces
		2 tests from My/DerivedTest/0

		2 tests from My/DerivedTest/0 (1 ms total)
		[----------] 
		2 tests from My/DerivedTest/1
		, where TypeParam = long
		[ RUN      ] 
		My/DerivedTest/1.ValuesAreCorrect

		[       OK ] 
		My/DerivedTest/1.ValuesAreCorrect
		 (0 ms)
		[ RUN      ] 
		My/DerivedTest/1.ValuesAreStillCorrect

		[       OK ] 
		My/DerivedTest/1.ValuesAreStillCorrect
		 (0 ms)
		[----------] 
		2 tests from My/DerivedTest/1 (1 ms total)
		[----------] 
		2 tests from Int/TypedTestP1/0
		, where TypeParam = int
		[ RUN      ] 
		Int/TypedTestP1/0.A

		[       OK ] 
		Int/TypedTestP1/0.A
		 (0 ms)
		[ RUN      ] 
		Int/TypedTestP1/0.B

		[       OK ] 
		Int/TypedTestP1/0.B
		 (0 ms)
		[----------] 
		2 tests from Int/TypedTestP1/0 (1 ms total)
		[----------] 
		1 test from Int/TypedTestP2/0
		, where TypeParam = int
		[ RUN      ] 
		Int/TypedTestP2/0.A

		[       OK ] 
		Int/TypedTestP2/0.A
		 (0 ms)
		[----------] 
		1 test from Int/TypedTestP2/0 (0 ms total)
		[----------] 
		1 test from Double/TypedTestP2/0
		, where TypeParam = double
		[ RUN      ] 
		Double/TypedTestP2/0.A

		[       OK ] 
		Double/TypedTestP2/0.A
		 (0 ms)
		[----------] 
		1 test from Double/TypedTestP2/0 (0 ms total)
		[----------] 
		2 tests from My/ContainerTest/0
		, where TypeParam = class std::vector<double,class std::allocator<double> >
		[ RUN      ] 
		My/ContainerTest/0.CanBeDefaultConstructed

		[       OK ] 
		My/ContainerTest/0.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		My/ContainerTest/0.InitialSizeIsZero

		[       OK ] 
		My/ContainerTest/0.InitialSizeIsZero
		 (0 ms)
		[----------] 
		2 tests from My/ContainerTest/0 (1 ms total)
		[----------] 
		2 tests from My/ContainerTest/1
		, where TypeParam = class std::set<char,struct std::less<char>,class std::allocator<char> >
		[ RUN      ] 
		My/ContainerTest/1.CanBeDefaultConstructed

		[       OK ] 
		My/ContainerTest/1.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		My/ContainerTest/1.InitialSizeIsZero

		[       OK ] 
		My/ContainerTest/1.InitialSizeIsZero
		 (0 ms)
		[----------] 
		2 tests from My/ContainerTest/1 (1 ms total)
		[----------] 
		2 tests from My/NumericTest/0
		, where TypeParam = int
		[ RUN      ] 
		My/NumericTest/0.DefaultIsZero

		[       OK ] 
		My/NumericTest/0.DefaultIsZero
		 (0 ms)
		[ RUN      ] 
		My/NumericTest/0.ZeroIsLessThanOne

		[       OK ] 
		My/NumericTest/0.ZeroIsLessThanOne
		 (0 ms)
		[----------] 
		2 tests from My/NumericTest/0 (1 ms total)
		[----------] 
		2 tests from My/NumericTest/1
		, where TypeParam = double
		[ RUN      ] 
		My/NumericTest/1.DefaultIsZero

		[       OK ] 
		My/NumericTest/1.DefaultIsZero
		 (0 ms)
		[ RUN      ] 
		My/NumericTest/1.ZeroIsLessThanOne

		[       OK ] 
		My/NumericTest/1.ZeroIsLessThanOne
		 (0 ms)
		[----------] 
		2 tests from My/NumericTest/1 (0 ms total)
		[----------] 
		5 tests from My/TrimmedTest/0
		, where TypeParam = int
		[ RUN      ] 
		My/TrimmedTest/0.Test1

		[       OK ] 
		My/TrimmedTest/0.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test2

		[       OK ] 
		My/TrimmedTest/0.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test3

		[       OK ] 
		My/TrimmedTest/0.Test3
		 (1 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test4

		[       OK ] 
		My/TrimmedTest/0.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test5

		[       OK ] 
		My/TrimmedTest/0.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/0 (1 ms total)
		[----------] 
		5 tests from My/TrimmedTest/1
		, where TypeParam = double
		[ RUN      ] 
		My/TrimmedTest/1.Test1

		[       OK ] 
		My/TrimmedTest/1.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test2

		[       OK ] 
		My/TrimmedTest/1.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test3

		[       OK ] 
		My/TrimmedTest/1.Test3
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test4

		[       OK ] 
		My/TrimmedTest/1.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test5

		[       OK ] 
		My/TrimmedTest/1.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/1 (1 ms total)
		[----------] 
		5 tests from My/TrimmedTest/2
		, where TypeParam = struct library2::MyPair<int,int>
		[ RUN      ] 
		My/TrimmedTest/2.Test1

		[       OK ] 
		My/TrimmedTest/2.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test2

		[       OK ] 
		My/TrimmedTest/2.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test3

		[       OK ] 
		My/TrimmedTest/2.Test3
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test4

		[       OK ] 
		My/TrimmedTest/2.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test5

		[       OK ] 
		My/TrimmedTest/2.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/2 (2 ms total)
		[----------] 
		2 tests from Vector/ContainerTest/0
		, where TypeParam = class std::vector<int,class std::allocator<int> >
		[ RUN      ] 
		Vector/ContainerTest/0.CanBeDefaultConstructed

		[       OK ] 
		Vector/ContainerTest/0.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		Vector/ContainerTest/0.InitialSizeIsZero

		[       OK ] 
		Vector/ContainerTest/0.InitialSizeIsZero
		 (0 ms)
		[----------] 
		2 tests from Vector/ContainerTest/0 (0 ms total)
		[----------] 
		Global test environment tear-down
		[==========] 
		239 tests from 46 test cases ran.
		 (170 ms total)

		[  PASSED  ] 
		239 tests.
		Connect to pipe \SEVPipe\9021f601f657ec5f
		[==========] 
		Running 241 tests from 48 test cases.
		[----------] 
		Global test environment set-up.
		[----------] 
		1 test from TestPartResultArrayDeathTest

		[ RUN      ] 
		TestPartResultArrayDeathTest.DiesWhenIndexIsOutOfBound

		[       OK ] 
		TestPartResultArrayDeathTest.DiesWhenIndexIsOutOfBound
		 (0 ms)
		[----------] 
		1 test from TestPartResultArrayDeathTest (0 ms total)
		[----------] 
		4 tests from TypedTestCasePStateDeathTest

		[ RUN      ] 
		MessageTest.DefaultConstructor
		MessageTest.StreamsFloat
		MessageTest.StreamsPointer
		MessageTest.StreamsCString
		 (0 ms)

		MessageTest.GetString
		 (0 ms)
		 (0 ms)

		EXPECT_PRED1Test.FunctorOnUserTypeSuccess
		[ RUN      ] 
		8 tests from EXPECT_PRED1Test (5 ms total)
		 (0 ms)

		ASSERT_PRED1Test.FunctionOnUserTypeFailure
		ASSERT_PRED1Test.FunctorOnUserTypeFailure
		EXPECT_PRED_FORMAT1Test.FunctorOnBuiltInTypeSuccess
		 (1 ms)
		ASSERT_PRED_FORMAT1Test.FunctionOnBuiltInTypeSuccess

		ASSERT_PRED_FORMAT1Test.FunctionOnUserTypeFailure
		8 tests from EXPECT_PRED2Test
		 (0 ms)
		[ RUN      ] 
		 (1 ms)

		8 tests from EXPECT_PRED2Test (6 ms total)
		8 tests from ASSERT_PRED2Test

		[       OK ] 
		ASSERT_PRED2Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		 (0 ms)

		[       OK ] 
		ASSERT_PRED2Test.FunctionOnBuiltInTypeFailure
		 (1 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctionOnUserTypeFailure

		[       OK ] 
		ASSERT_PRED2Test.FunctionOnUserTypeFailure
		 (1 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctorOnBuiltInTypeFailure

		[       OK ] 
		ASSERT_PRED2Test.FunctorOnBuiltInTypeFailure
		 (1 ms)
		[ RUN      ] 
		ASSERT_PRED2Test.FunctorOnUserTypeFailure
		ASSERT_PRED_FORMAT2Test.FunctionOnBuiltInTypeFailure
		[ RUN      ] 
		EXPECT_PRED3Test.FunctionOnBuiltInTypeFailure
		EXPECT_PRED3Test.FunctorOnUserTypeFailure

		[       OK ] 
		EXPECT_PRED3Test.FunctorOnUserTypeFailure
		8 tests from EXPECT_PRED3Test (7 ms total)

		[       OK ] 
		ASSERT_PRED3Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED3Test.FunctionOnUserTypeSuccess
		ASSERT_PRED3Test.FunctorOnBuiltInTypeSuccess

		ASSERT_PRED3Test.FunctionOnBuiltInTypeFailure
		[ RUN      ] 
		[ RUN      ] 
		8 tests from ASSERT_PRED3Test (7 ms total)

		[       OK ] 
		EXPECT_PRED_FORMAT3Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		EXPECT_PRED_FORMAT3Test.FunctorOnBuiltInTypeSuccess
		[ RUN      ] 

		 (1 ms)
		 (1 ms)
		8 tests from EXPECT_PRED_FORMAT3Test (5 ms total)
		[----------] 

		 (0 ms)

		ASSERT_PRED_FORMAT3Test.FunctorOnUserTypeFailure
		8 tests from ASSERT_PRED_FORMAT3Test (5 ms total)
		[----------] 
		8 tests from EXPECT_PRED4Test

		[ RUN      ] 
		EXPECT_PRED4Test.FunctionOnBuiltInTypeSuccess

		[       OK ] 
		EXPECT_PRED4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		EXPECT_PRED4Test.FunctionOnUserTypeSuccess

		[       OK ] 
		EXPECT_PRED4Test.FunctionOnUserTypeSuccess
		 (0 ms)
		[       OK ] 
		 (1 ms)
		EXPECT_PRED4Test.FunctorOnBuiltInTypeFailure

		[       OK ] 
		[ RUN      ] 
		EXPECT_PRED4Test.FunctorOnUserTypeFailure

		[       OK ] 
		EXPECT_PRED4Test.FunctorOnUserTypeFailure
		 (1 ms)
		[----------] 
		8 tests from EXPECT_PRED4Test (9 ms total)
		[----------] 
		8 tests from ASSERT_PRED4Test

		[ RUN      ] 
		ASSERT_PRED4Test.FunctionOnBuiltInTypeSuccess

		[       OK ] 
		ASSERT_PRED4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED4Test.FunctionOnUserTypeSuccess

		[       OK ] 
		ASSERT_PRED4Test.FunctionOnUserTypeSuccess
		 (0 ms)
		ASSERT_PRED4Test.FunctorOnBuiltInTypeSuccess
		[ RUN      ] 
		[       OK ] 
		ASSERT_PRED4Test.FunctorOnBuiltInTypeFailure

		8 tests from ASSERT_PRED4Test (9 ms total)
		[----------] 

		[       OK ] 
		EXPECT_PRED_FORMAT4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		[       OK ] 
		EXPECT_PRED_FORMAT4Test.FunctorOnUserTypeSuccess
		[       OK ] 
		EXPECT_PRED_FORMAT4Test.FunctionOnUserTypeFailure
		[ RUN      ] 
		8 tests from EXPECT_PRED_FORMAT4Test (6 ms total)
		[----------] 
		[ RUN      ] 

		[       OK ] 
		ASSERT_PRED_FORMAT4Test.FunctionOnBuiltInTypeSuccess
		 (0 ms)
		 (0 ms)

		ASSERT_PRED_FORMAT4Test.FunctionOnBuiltInTypeFailure
		[       OK ] 
		 (1 ms)
		[----------] 
		[ RUN      ] 
		EXPECT_PRED5Test.FunctorOnBuiltInTypeSuccess


		 (2 ms)
		EXPECT_PRED5Test.FunctorOnUserTypeFailure
		[ RUN      ] 

		ASSERT_PRED5Test.FunctorOnBuiltInTypeSuccess
		 (0 ms)
		[ RUN      ] 
		ASSERT_PRED5Test.FunctorOnBuiltInTypeFailure
		ASSERT_PRED5Test.FunctorOnUserTypeFailure
		EXPECT_PRED_FORMAT5Test.FunctionOnBuiltInTypeSuccess
		[       OK ] 
		EXPECT_PRED_FORMAT5Test.FunctorOnBuiltInTypeSuccess

		EXPECT_PRED_FORMAT5Test.FunctionOnUserTypeFailure
		EXPECT_PRED_FORMAT5Test.FunctorOnUserTypeFailure
		[----------] 
		ASSERT_PRED_FORMAT5Test.FunctionOnBuiltInTypeSuccess
		ASSERT_PRED_FORMAT5Test.FunctionOnUserTypeSuccess
		[ RUN      ] 
		[ RUN      ] 
		ASSERT_PRED_FORMAT5Test.FunctionOnUserTypeFailure
		ASSERT_PRED_FORMAT5Test.FunctorOnBuiltInTypeFailure
		[----------] 
		[       OK ] 

		10 tests from TestPartResultTest
		TestPartResultTest.ResultAccessorsWork
		TestPartResultTest.type
		[ RUN      ] 

		 (0 ms)
		TestPartResultTest.FatallyFailed
		[----------] 
		TestPartResultArrayTest.InitialSizeIsZero
		[ RUN      ] 
		2 tests from CommonTest/0
		 (0 ms)
		2 tests from CommonTest/0 (1 ms total)
		[       OK ] 
		 (0 ms)
		[       OK ] 
		, where TypeParam = int
		[----------] 
		[       OK ] 

		2 tests from TypedTestCasePStateTest
		TypedTestCasePStateTest.IgnoresOrderAndSpaces
		2 tests from My/DerivedTest/0
		[ RUN      ] 
		2 tests from My/DerivedTest/0 (1 ms total)
		[----------] 
		2 tests from My/DerivedTest/1
		, where TypeParam = long
		[ RUN      ] 
		My/DerivedTest/1.ValuesAreCorrect

		[       OK ] 
		My/DerivedTest/1.ValuesAreCorrect
		 (0 ms)
		[ RUN      ] 
		My/DerivedTest/1.ValuesAreStillCorrect

		[       OK ] 
		My/DerivedTest/1.ValuesAreStillCorrect
		 (0 ms)
		[----------] 
		2 tests from My/DerivedTest/1 (1 ms total)
		[----------] 
		2 tests from Int/TypedTestP1/0
		, where TypeParam = int
		[ RUN      ] 
		Int/TypedTestP1/0.A

		[       OK ] 
		Int/TypedTestP1/0.A
		 (0 ms)
		[ RUN      ] 
		Int/TypedTestP1/0.B

		[       OK ] 
		Int/TypedTestP1/0.B
		 (0 ms)
		[----------] 
		2 tests from Int/TypedTestP1/0 (1 ms total)
		[----------] 
		1 test from Int/TypedTestP2/0
		, where TypeParam = int
		[ RUN      ] 
		Int/TypedTestP2/0.A

		[       OK ] 
		Int/TypedTestP2/0.A
		 (0 ms)
		[----------] 
		1 test from Int/TypedTestP2/0 (0 ms total)
		[----------] 
		1 test from Double/TypedTestP2/0
		, where TypeParam = double
		[ RUN      ] 
		Double/TypedTestP2/0.A

		[       OK ] 
		Double/TypedTestP2/0.A
		 (0 ms)
		[----------] 
		1 test from Double/TypedTestP2/0 (0 ms total)
		[----------] 
		2 tests from My/ContainerTest/0
		, where TypeParam = class std::vector<double,class std::allocator<double> >
		[ RUN      ] 
		My/ContainerTest/0.CanBeDefaultConstructed

		[       OK ] 
		My/ContainerTest/0.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		My/ContainerTest/0.InitialSizeIsZero

		[       OK ] 
		My/ContainerTest/0.InitialSizeIsZero
		 (1 ms)
		[----------] 
		2 tests from My/ContainerTest/0 (1 ms total)
		[----------] 
		2 tests from My/ContainerTest/1
		, where TypeParam = class std::set<char,struct std::less<char>,class std::allocator<char> >
		[ RUN      ] 
		My/ContainerTest/1.CanBeDefaultConstructed

		[       OK ] 
		My/ContainerTest/1.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		My/ContainerTest/1.InitialSizeIsZero

		[       OK ] 
		My/ContainerTest/1.InitialSizeIsZero
		 (1 ms)
		[----------] 
		2 tests from My/ContainerTest/1 (1 ms total)
		[----------] 
		2 tests from My/NumericTest/0
		, where TypeParam = int
		[ RUN      ] 
		My/NumericTest/0.DefaultIsZero

		[       OK ] 
		My/NumericTest/0.DefaultIsZero
		 (0 ms)
		[ RUN      ] 
		My/NumericTest/0.ZeroIsLessThanOne

		[       OK ] 
		My/NumericTest/0.ZeroIsLessThanOne
		 (0 ms)
		[----------] 
		2 tests from My/NumericTest/0 (1 ms total)
		[----------] 
		2 tests from My/NumericTest/1
		, where TypeParam = double
		[ RUN      ] 
		My/NumericTest/1.DefaultIsZero

		[       OK ] 
		My/NumericTest/1.DefaultIsZero
		 (0 ms)
		[ RUN      ] 
		My/NumericTest/1.ZeroIsLessThanOne

		[       OK ] 
		My/NumericTest/1.ZeroIsLessThanOne
		 (0 ms)
		[----------] 
		2 tests from My/NumericTest/1 (0 ms total)
		[----------] 
		5 tests from My/TrimmedTest/0
		, where TypeParam = int
		[ RUN      ] 
		My/TrimmedTest/0.Test1

		[       OK ] 
		My/TrimmedTest/0.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test2

		[       OK ] 
		My/TrimmedTest/0.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test3

		[       OK ] 
		My/TrimmedTest/0.Test3
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test4

		[       OK ] 
		My/TrimmedTest/0.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/0.Test5

		[       OK ] 
		My/TrimmedTest/0.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/0 (1 ms total)
		[----------] 
		5 tests from My/TrimmedTest/1
		, where TypeParam = double
		[ RUN      ] 
		My/TrimmedTest/1.Test1

		[       OK ] 
		My/TrimmedTest/1.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test2

		[       OK ] 
		My/TrimmedTest/1.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test3

		[       OK ] 
		My/TrimmedTest/1.Test3
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test4

		[       OK ] 
		My/TrimmedTest/1.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/1.Test5

		[       OK ] 
		My/TrimmedTest/1.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/1 (2 ms total)
		[----------] 
		5 tests from My/TrimmedTest/2
		, where TypeParam = struct library2::MyPair<int,int>
		[ RUN      ] 
		My/TrimmedTest/2.Test1

		[       OK ] 
		My/TrimmedTest/2.Test1
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test2

		[       OK ] 
		My/TrimmedTest/2.Test2
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test3

		[       OK ] 
		My/TrimmedTest/2.Test3
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test4

		[       OK ] 
		My/TrimmedTest/2.Test4
		 (0 ms)
		[ RUN      ] 
		My/TrimmedTest/2.Test5

		[       OK ] 
		My/TrimmedTest/2.Test5
		 (0 ms)
		[----------] 
		5 tests from My/TrimmedTest/2 (1 ms total)
		[----------] 
		2 tests from Vector/ContainerTest/0
		, where TypeParam = class std::vector<int,class std::allocator<int> >
		[ RUN      ] 
		Vector/ContainerTest/0.CanBeDefaultConstructed

		[       OK ] 
		Vector/ContainerTest/0.CanBeDefaultConstructed
		 (0 ms)
		[ RUN      ] 
		Vector/ContainerTest/0.InitialSizeIsZero

		[       OK ] 
		Vector/ContainerTest/0.InitialSizeIsZero
		 (0 ms)
		[----------] 
		2 tests from Vector/ContainerTest/0 (0 ms total)
		[----------] 
		Global test environment tear-down
		[==========] 
		241 tests from 48 test cases ran.
		 (172 ms total)

		[  PASSED  ] 
		241 tests.

**Pull request closed without merging**